src/custom-elements/src/components/popover/popover.tsx

```diff
@@ -375,8 +390,70 @@ export class Popover {
     };
     this._configurePopoverPopper();
     this._componentLoaded = true;
-    document.addEventListener('click', this._documentClickHandler);
-    document.addEventListener('keyup', this._documentKeyHandler);
+
+    if (this.drag) {
```

Comment:
I would include this code into a separate initializing function. I would also create a function for disposing the draggable feature. I would call the initialize and dispose on the change event of the `drag` prop. Please, ensure that it is not called twice. I would also call the dispose function on the dispose of the main component.

Reply:
You might even want to create a draggable class that holds the functionality.

Reply:
Done, great idea, thanks 😉